test/trustedresources.go

```diff
@@ -124,44 +115,184 @@ func SetupTrustedResourceConfig(ctx context.Context, keypath string, resourceVer
 	return store.ToContext(ctx)
 }
 
-// GetSignerFromFile generates key files to tmpdir, return signer and pubkey path
-func GetSignerFromFile(ctx context.Context, t *testing.T) (signature.Signer, string, error) {
+// SetupVerificationPolicies set verification policies
```

Comment:
comments that just repeat the name of the function don't really add much value, it'd be better to add detail about what the function is doing (e.g. what the purpose is any arguments, what it returns)

more on this at https://go.dev/doc/comment

in particular for this function, im interested in things like:
- why it returns a k8s client set and what that clientset is for
- what kind of verification policies it sets up